etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamEndpointClient.java

```diff
@@ -109,14 +109,14 @@ public String requestWatersEndpoint(String body, String bearerToken)
     }
 
     @Override
-    public String requestHistoryEndpoint(String submissionId, String bearerToken)
+    public String requestHistoryEndpoint(String outboundReportId, String bearerToken)
```

Comment:
Looks good. I think you missed the `reportId` in `requestDeliveryEndpoint` below to be renamed to `inboundReportId`